packages/rune-games-cli/src/flows/List/Details.tsx

```diff
@@ -0,0 +1,37 @@
+import { Text, Box } from "ink"
+import React from "react"
+
+import { Step } from "../../components/Step.js"
+import { useGame } from "../../gql/useGame.js"
+import { useMe } from "../../gql/useMe.js"
+
+export function Details({ gameId }: { gameId: number }) {
+  const { me } = useMe()
+  const { game, gameLoading } = useGame(gameId)
+  const gameDevs = game?.gameDevs.nodes
+  const gameDevMe = gameDevs?.find((gameDev) => gameDev.userId === me?.devId)
+
+  if (gameLoading) {
+    return <Step status="waiting" label="Loading game" />
```

Comment:
Initially it seemed strange to have a waiting status in `Details` and also in `List`, but I see that the labels are different. Unless the intent is that this `Details` component will be reused somewhere else outside of `List`, it might be nice to centralize all the "loading" status displays in one place (`List`) instead of two.

Reply:
Done in https://github.com/rune/rune/pull/292/commits/ff22bccf453cd1423dad25e998e6b233f53f0e68